mad_spark_multiagent/run_madspark.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Run MadSpark with environment variables loaded
+
+# Activate virtual environment
+source venv/bin/activate
```

Shellcheck (0.10.0)

[info] Not following: venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)